Source/driver/Castro_util.F90

```diff
@@ -679,6 +680,64 @@ end function area
 
 
 
+  function dLogArea(i, j, k, dir) result(dlogarear)
+    ! Given an index (i,j,k) and a direction dir, return the
+    ! coefficient that appears in the geometry source terms (only
+    ! relevant for non-Cartesian geometries).
+
+    use amrinfo_module, only: amr_level
+    use amrex_constants_module, only: ZERO, ONE, TWO
+    use prob_params_module, only: dim, coord_type, dx_level
+#ifndef AMREX_USE_CUDA
+    use castro_error_module, only: castro_error
+#endif
+    use amrex_fort_module, only: rt => amrex_real
+
+    implicit none
+
+    integer, intent(in) :: i, j, k, dir
+
+    real(rt) :: dlogarear
+
+    logical :: cc(3)
+    real(rt) :: dx(3), loc(3)
+
+    !$gpu
+
+    cc(:) = .true.
+
+    dx = dx_level(:,amr_level)
+
+    dlogarear = ZERO
+
+    if (coord_type .eq. 1 .and. dir .eq. 1) then
+
+       ! Cylindrical (2D only)
+
+       ! Get edge-centered position
+
+       loc = position(i,j,k,cc(1),cc(2),cc(3))
```

Comment:
don't you need to do `cc(idir) = .false.` here?

Reply:
I would have thought so too, but I looked at the dLogArea calculation in AMReX and it seems to be using cell-centered indexing, which I think is consistent with how we're computing the geometry source terms.

Reply:
I'll fix the comment to be consistent.